pulsar-client-cpp/include/pulsar/Schema.h

```diff
@@ -143,6 +162,14 @@ class PULSAR_PUBLIC SchemaInfo {
     SchemaInfo(SchemaType schemaType, const std::string &name, const std::string &schema,
                const StringMap &properties = StringMap());
 
+    /**
+     * @param keySchema  the key schema.
+     * @param valueSchema  the value schema.
+     * @param keyValueEncodingType Encoding types of supported KeyValueSchema for Pulsar messages.
+     */
+    SchemaInfo(const SchemaInfo &keySchema, const SchemaInfo &valueSchema,
+               const KeyValueEncodingType &keyValueEncodingType);
```

Comment:
I think we can add a default value (`INLINE`) for it.

```suggestion
    SchemaInfo(const SchemaInfo &keySchema, const SchemaInfo &valueSchema,
               const KeyValueEncodingType &keyValueEncodingType = KeyValueEncodingType::INLINE);
```